editor/find_in_files.cpp

```diff
@@ -702,21 +657,24 @@ void FindInFilesPanel::_notification(int p_what) {
 	}
 }
 
-void FindInFilesPanel::_on_result_found(String fpath, int line_number, int begin, int end, String text) {
+void FindInFilesPanel::_on_result_found(String display_text, String relative_file_path, String absolute_file_path, int line_number, int begin, int end, String text) {
 	TreeItem *file_item;
-	HashMap<String, TreeItem *>::Iterator E = _file_items.find(fpath);
+	HashMap<String, TreeItem *>::Iterator E = _file_items.find(relative_file_path);
```

Comment:
Why are we using relative paths here? Aren't they by definition prone to collisions?

Reply:
Good point, gonna fix it!